src/install.js

```diff
@@ -14,4 +24,7 @@ module.exports = function (api) {
   api.extendJsonFile('.vscode/extensions.json', {
     recommendations: ['apollographql.vscode-apollo'],
   })
+
+  // We extend /quasar.conf.js
+  api.extendQuasarConf(extendConf)
```

Comment:
This is not needed and will not do anything. Install only runs once, but we need the boot file on every run, which is already handled in the `index.js` script. It also handles SSR and such. So, please remove it.

Reply:
@efonte - Can you reply/ take action please?

Scott